_quarto.yml

```diff
@@ -0,0 +1,34 @@
+project:
+  type: book
+  # pre-render: quarto render slides/*.qmd
+  # resources:
+  #   - slides/*.html
+
+book:
+  title: "Mediation course"
+  author: 
+    - Jie Zhang
+    - Daniel Witte
+  date: last-modified
+  repo-url: https://github.com/steno-aarhus/IDEG-mediation-course
+  site-url: https://steno-aarhus.github.io/IDEG-mediation-course
+  chapters:
+    - index.qmd
+    - part: "Preamble"
+    - part: "Sessions"
+      chapters:
+      - sessions/welcome.qmd
+      - sessions/motivation.qmd
+      - sessions/traditional-mediation-analysis.qmd
+  appendices:
+    - appendix/welcome.qmd
```

Comment:
This section is where you add the files you want put onto the website. Unfortunately, you can't put slide files here. But we deal with that when you get to it.

I didn't put anything into `part: "Preamble"` because your `precourse.qmd` file was in one of the `_files/` folders, which I moved in #2, so that needs to be merged first before you can add it correctly here.